OPHD/UI/PopulationPanel.cpp

```diff
@@ -65,7 +66,7 @@ PopulationPanel::PopulationPanel() :
 		imageCache.load("ui/skin/window_bottom_right.png")
 	}
 {
-	constexpr int linesOfText = 14;
+	constexpr int linesOfText = 16;
```

Comment:
![PopPanel](https://github.com/OutpostUniverse/OPHD/assets/5853140/9ffba6d7-cf1b-46fe-9e84-0dc75fcb8d7e)

I do not know if this helps. But I made a quick visualizer for the 16 lines of text in red.